package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "netlify-cms",
-  "version": "1.3.1",
-  "description": "Netlify CMS lets content editors work on structured content stored in git",
+  "name": "extinguisher",
+  "version": "1.4.0",
```

Comment:
This should work fine for now, but we might want the version number to match the revision of the main project we're on.  We'll see what happens from here.

Reply:
You're right. I will change it. Regardless, such a small change should not be a whole new minor release version.